codespell_lib/data/dictionary.txt

```diff
@@ -18282,7 +18284,9 @@ nighboring->neighboring
 nightfa;;->nightfall
 nightime->nighttime
 nimutes->minutes
+nin->min
```

Comment:
Should probably also suggest `bin` and `nine` ? Possibly also `inn`.

Reply:
Done.